TODO

```diff
@@ -0,0 +1 @@
+read: https://travisofthenorth.com/blog/2017/2/2/postgres-adding-foreign-keys-with-zero-downtime
```

Comment:
This file was probably added by accident in the last WIP commit. Will get it out when finalizing the work here.